rsyslog/tasks/main.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: include OS-specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_os_family }}-{{ ansible_distribution_major_version }}.yml"
+    - "{{ ansible_os_family }}.yml"
+
+- name: install rsyslog package
+  package:
+    name: rsyslog
+    state: installed
+
+- name: configure rsyslog
+  template:
+    src: rsyslog.conf.j2
+    dest: /etc/rsyslog.conf
+    mode: 0644
+    owner: root
+    group: root
```

Comment:
Should group be like this ( in other places in this file too)?
    group: "{{ ansible_wheel }}"